### PR TITLE
Feature: non-article views capture (attempt 2)

### PR DESCRIPTION
**Pull request by lsh-0**
This doesn't handle non-article page views further back than 2017-06-01 (launch of 2.0). There is room for this, just not in this PR.

### CODE REVIEW

File: src/nametrics/apps.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
```
[the remark this answers is not in the document]
Comment (giorgiosironi) [reply src/nametrics/apps.py]: Asked around in https://elifesciences.slack.com/archives/C025LBBR4/p1526402898000033

[the remark this answers is not in the document]
Comment (lsh-0) [reply src/nametrics/apps.py]: suggestion: https://github.com/elifesciences/elife-metrics/pull/62

[the remark this answers is not in the document]
Comment (giorgiosironi) [reply src/nametrics/apps.py]: > Don't appeal to the 'average reader' though

Given the title of the pull request it didn't take long to figure out what the acronym was standing for, but I think of anyone (me, you, @seanwiseman, etc) looking inside the code a while from now for a bug or an extension down the road; I often overestimate how much I can remember

[the remark this answers is not in the document]
Comment (lsh-0) [reply src/nametrics/apps.py]: sure, I think ever coder can appreciate that.

**Pull request merged**